docs/trace/tail-based-sampling-span-level/README.md

```diff
@@ -0,0 +1,103 @@
+# Tail Based Sampling at a span level: An Example
+
+This is an example / proof of concept to achieve tail-based sampling at a
+span level using the extensibility mechanisms in OpenTelemetry.NET.
+
+This is a way to achieve a combination of:
+
+- Head-based sampling (based on probabilistic sampling) and
+- Tail-based sampling to get all failure spans (non-probabilistic sampling).
+
+## How does this sampling example work?
+
+We use a hybrid approach: we do head based sampling to get a
+probabilistic subset of all spans which includes both successful spans
```

Comment:
This doc is using the words Span and SpanProcessor. I'd suggest to stick with Activity and ActivityProcessor, but the very first occurrence include Span,SPanProcessor in brackets, and then just stick with Activity terminology